web/ViewModels/AdminListModel.cs

```diff
@@ -9,7 +9,6 @@ public AdminListModel(List<UserDbModel> currentAdmins, List<UserDbModel> pending
         CurrentAdmins = currentAdmins;
         PendingAdmins = pendingAdmins;
     }
-

```

Comment:
We should keep this newline but make sure it doesn't have any trailing spaces on it 🙂

Reply:
Oh, that makes more sense - I was a bit confused by that change.